javascriptv3/example_code/README.md

```diff
@@ -1,35 +1,44 @@
-* Create a shared configurations file with your user credentials. See the [AWS SDK for JavaScript v3 Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/loading-node-credentials-shared.html) for more information.
+## Types of code examples
+* **Single-service actions** - Code examples that show you how to call individual service functions.
 
-## Running the examples
-Examples are written for either execution in a browser script or in Node.js, usually depending on the use case for each. The two scenarios typically differ in how you supply credentials to the code. For information on the differences, see the [AWS SDK for JavaScript v3 Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/setting-credentials.html).
+* **Single-service scenarios** - Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
 
-The Node.js examples are all run from the command line.
+* **Cross-service examples** - Sample applications that work across multiple AWS services.
 
-## Unit tests
-[Unit tests](./tests) are provided for many examples, using the [Jest](https://jestjs.io/) framework.
+### Find code examples
+Single-service actions and scenarios are organized by AWS service in this folder. A README in each folder lists and describes how to run the examples.
 
-To run tests, navigate to the **tests** folder.
+Cross-service examples are located in the [*cross-services folder*](./cross-services). A README in each folder describes how to run the example.
 
-For example, to run tests on the S3 folder, enter the following sequence of commands at the command prompt:
+## ⚠️ Important
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account. 
+* We recommend that you grant your code the least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
-```
-npm install node -g
-cd javascriptv3/example_code/s3/tests
-npm install
-npm test
-```
-## Resources
+
+### Prerequisites
+* Latest stable version of Node.js
+* A shared configuration file with your user credentials. See the [AWS SDK for JavaScript v3 Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/loading-node-credentials-shared.html) for more information.
```

Comment:
Update second sentence in this bullet point: 
For more information, see the [AWS SDK for JavaScript (v3) Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/loading-node-credentials-shared.html).